src/Components/LoginPage/_LoginPage.scss

```diff
@@ -66,5 +66,10 @@ $text-color: #D9D9D9;
             }
 
         }
+
+        p {
+            font-size: 1.75vw;
+            color: #44c4a1;
```

Comment:
I see color names at the top being used, would recommend also adding this as a color for reference (Ex. $login-area-text etc.)

Reply:
That is a great catch! I will change the color to be assigned to a variable like the variables for the colors at the top of this file!